haskell/test/SMT.hs

```diff
@@ -710,25 +709,22 @@ applyTransfer ccSender storage tis = do
   foldl' (applySingleTransfer ccSender) (Right storage) tis
 
 applySingleTransfer :: Address -> Either ModelError SimpleStorage -> FA2.TransferParam -> Either ModelError SimpleStorage
-applySingleTransfer ccSender estorage
-  (arg #from_ -> from, arg #txs -> toItems)
-  = case estorage of
-      Left err -> Left err
-      Right storage@(SimpleStorage {..}) ->
-        if ccSender == from || isOperatorOf storage ccSender from
-          then let
-            in foldl' singleTranferTx (Right storage) toItems
-          else Left FA2_NOT_OPERATOR
+applySingleTransfer ccSender estorage (FA2.TransferParam from txs) =
+  case estorage of
+    Left err -> Left err
+    Right storage@(SimpleStorage {..}) ->
+      if ccSender == from || isOperatorOf storage ccSender from
+        then let
+          in foldl' singleTranferTx (Right storage) txs
```

Comment:
You didn't change it in this PR, but I am looking at it now and don't understand this construct. AFAIU there is empty `let in` block which is redundant. If it is, can you remove it please while we are at it?

Reply:
Done